dev-embedded/ponyprog/ponyprog-3.0.0.ebuild

```diff
@@ -0,0 +1,42 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit cmake-utils desktop
+
+DESCRIPTION="Serial device programmer"
+HOMEPAGE="https://github.com/lancos/ponyprog/"
+SRC_URI="https://github.com/lancos/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="LGPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+
+RDEPEND="dev-qt/qtcore:5
+	dev-qt/qtgui:5
+	dev-qt/qtmultimedia:5
+	dev-qt/qtprintsupport:5
+	app-editors/qhexedit2:0="
```

Comment:
Alphabetical order, please.

For future work you may do: It's easier to read when dependencies are grouped by unversioned, versioned, and use-conditional.

Reply:
done